recipes-core/glibc/glibc/cp936-gconv-modules.patch

```diff
@@ -1,16 +1,6 @@
-From a33deff1b9fc055aa9cfc6b1569cb218742637d2 Mon Sep 17 00:00:00 2001
-From: Bill Pittman <bill.pittman@ni.com>
-Date: Fri, 19 Jun 2020 14:15:16 -0500
-
----
- iconvdata/gconv-modules | 9 ++++++---
- 1 file changed, 6 insertions(+), 3 deletions(-)
-
-diff --git a/iconvdata/gconv-modules b/iconvdata/gconv-modules
-index 16d09eb98d..a1279c8719 100644
```

Comment:
Was removal of these lines intentional? I notice you do on some of the patches but not others.

Reply:
This was not intentional.. let me take a look.

Reply:
These changes came directly from sumo version of these files which didn't have these comments.